site/site/cli/usage/configuration.md

```diff
@@ -24,3 +24,5 @@ The available log levels:
 <!-- TODO: config link -->
 
 Taplo CLI by default searches for a Taplo config file in the current working directory, this behaviour can be disabled by either supplying `--no-auto-config` or `--config <path>` flags.
+
+If a config file is not provided and no file is found in the current working directory, Taplo CLI will try to retrieve the configuration file path from the `TAPLO_CONFIG` environment variable.
```

Comment:
Let's remove this and instead update the line above to say that it is also possible to set the `TAPLO_CONFIG` environment variable (the `--config` flag taking precedence).